lib/lib-table-cluster/src/main/java/org/sagebionetworks/table/cluster/SQLTranslatorUtils.java

```diff
@@ -799,20 +876,22 @@ static void translateQueryFilters(StringBuilder builder, QueryFilter filter){
 	}
 
 	static void translateSingleValueFilters(StringBuilder builder, ColumnSingleValueQueryFilter filter){
+		ValidateArgument.requiredNotEmpty(filter.getColumnName(), "ColumnSingleValueQueryFilter.columnName");
+		ValidateArgument.requiredNotEmpty(filter.getValues(), "ColumnSingleValueQueryFilter.likeValues");
 		ValidateArgument.required(filter.getOperator(), "ColumnSingleValueQueryFilter.operator");
 		switch (filter.getOperator()){
 			case LIKE:
 				appendLikeFilter(builder, filter);
 				break;
+			case HAS_LIKE:

```

Comment:
Maybe a new ColumnMultiValueFunctionQueryFilter with function name and parameters.